pkg/secrets/aws.go

```diff
@@ -0,0 +1,72 @@
+package secrets
+
+import (
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	// AWSSecretType represents the secret type for AWS credentials.
+	AWSSecretType string = "secrets.kanister.io/aws"
```

Comment:
[minor] do these need to be exported?

Reply:
Not needed right now. However I think some Kanister package sure can use this value to validate their own input.